adr/2022_02_21_using_cart_by_guest_and_logged_in_customer.md

```diff
@@ -0,0 +1,60 @@
+# Using cart by guest and logged in customer
+
+* Status: accepted
+* Date: 2022-02-22
+
+## Context and Problem Statement
+
+Cart and its processing is one of the key aspects of Sylius. It turned out that it has a vulnerability and there is
+possible for an anonymous user to retrieve or override the cart of logged in customer by using only its email.
```

Comment:
Can we really retrieve their cart? We can override it and then we may have access to it
```suggestion
possible for an anonymous user to override the cart of the logged-in customer by using only its email.
```